modules/phase_field/src/auxkernels/OutputRGB.C

```diff
@@ -0,0 +1,46 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "OutputRGB.h"
+#include "GrainTracker.h"
+#include "EulerAngleProvider.h"
+#include "Euler2RGB.h"
+
+template<>
+InputParameters validParams<OutputRGB>()
+{
+  InputParameters params = validParams<AuxKernel>();
+  params.addClassDescription("Output RGB representation of crystal orientation from user object to an AuxVariable. The entire domain must have the same crystal structure.");
+  MooseEnum sd_enum = MooseEnum("100=1 010=2 001=3", "001");
+  params.addParam<MooseEnum>("sd", sd_enum, "Reference sample direction");
+  MooseEnum structure_enum = MooseEnum("cubic=43 hexagonal=62 tetragonal=42 trigonal=32 orthorhombic=22 monoclinic=2 triclinic=1");
+  params.addRequiredParam<MooseEnum>("crystal_structure",structure_enum,"Crystal structure of the material");
+  params.addRequiredParam<UserObjectName>("euler_angle_provider", "Name of Euler angle provider user object");
+  params.addRequiredParam<UserObjectName>("GrainTracker_object", "The GrainTracker UserObject to get values from.");
```

Comment:
Most (all?) of our input file parameters use all lowercase letters, so IMO this should be called `grain_tracker_object` for consistency.